pages/services/kubernetes/1.0.0-1.9.3/cni/index.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: layout.pug
+navigationTitle: Container Networking Interface (CNI)
+title: Container Networking Interface (CNI) Support
+menuWeight: 100
+excerpt:
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes -->
+
+
+## Container Networking Interface (CNI) Support
+
+By default, Kubernetes on DC/OS uses DC/OS overlay network to launch kubernetes
+pods. However, one can use some other network by selecting the network provider
+at the time of the kubernetes package installation.
+
+This document uses
+[`Calico`](https://docs.projectcalico.org/v2.0/getting-started/mesos/installation/dc-os/)
+as a sample CNI network provider to demonstrate the steps required to use
+some other CNI provider with kubernetes on DC/OS.
+
+## Using Calico with Kubernetes on DC/OS (For Enterprise Only)
+
+The very first thing that you would have to do is to install Calico on DC/OS. There
```

Comment:
The first step would be to install Calico on DC/OS. There are two ways to install Calico on DC/OS.

Reply:
Done.